### DISCUSSION TITLE
Rebranding buymeacoffee.js

### DISCUSSION THREAD

**Discussion by warengonzaga** [warengonzaga/buymeacoffee.js#38] [Brainstorm]
Since my co-author @thomasbnt is currently working on the latest branding. I'll put this up here.
I want to keep the discussion open here about the future branding of the project as it grows.

Mentioning a few issues here to make it all connected.

**Comment by warengonzaga** [comment]
Mentioning issue #37 which is connected to the rebranding idea.
Honestly, can't find the original asset I made for this project.

**Reply by thomasbnt** [reply]
Good initiative to create discussions. 💪🏼
